<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Muehle Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelMsg">
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Message here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNote">
     <property name="text">
      <string>Your stone is red</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="view">
     <property name="minimumSize">
      <size>
       <width>455</width>
       <height>455</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>455</width>
       <height>455</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonShowNet">
       <property name="text">
        <string>Show Network Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
